src/main/java/io/vertx/mqtt/MqttServerOptions.java

```diff
@@ -100,6 +101,12 @@ public MqttServerOptions setHost(String host) {
   }
 
   @Override
+  public MqttServerOptions setClientAuth(ClientAuth clientAuth) {
+    super.setClientAuth(clientAuth);
+    return this;
+  }
```

Comment:
Now that client authentication can be enabled on the server side, what do you think to add a unit test around that ?